lib/sshnp.dart

```diff
@@ -177,6 +178,10 @@ class SSHNP {
       throw StateError('Cannot init() - already initialized');
     }
 
+    if(!(await atSignIsActivated(atClient, sshnpdAtSign))) {
+        throw ('sshnpd atSign $sshnpdAtSign does not exist');
```

Comment:
With your current implementation, it might exist, but not yet be onboarded

Reply:
The exception message was incorrect
https://github.com/atsign-foundation/sshnoports/pull/211/commits/e2fd4426a83396de5536c59ada6a55d4c3f552b4

Reply:
Oh, an atSign can have a `publickey` but not be onboarded?